Build broke due to external dependency changes

### What did you do?
Triggered a build with no code changes.

### What did you expect to happen?
Build passes.

### What actually happened?
[Build failed.](https://travis-ci.org/nordsoftware/react-foundation/builds/121088908)

### What version of this module are you using?
develop

---

I updated the react-foundation docs link https://github.com/nordsoftware/react-foundation/commit/5791aa32450195b2389ab7a7455904fbb2fc5870 

It's only a minor .md change, but an external dependency broke the build for Node.js 4.x (5.x passes): 
```bash
npm ERR! peerinvalid The package eslint@2.7.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer babel-eslint@5.0.4 wants eslint@<2.3.0
```
https://travis-ci.org/nordsoftware/react-foundation/jobs/121088914#L183

[eslint](http://eslint.org/) 2.7.0 was only released three days ago, so that looks like the trigger.

We have:
```json
    "babel-eslint": "^5.0.0",
    "eslint": "^2.3.0",
```
https://github.com/nordsoftware/react-foundation/blob/develop/package.json#L57-L68

--- a/package.json
+++ b/package.json
@@ -54,7 +54,7 @@
   "devDependencies": {
     "babel-cli": "^6.6.5",
     "babel-core": "^6.7.0",
-    "babel-eslint": "^5.0.0",
+    "babel-eslint": "^6.0.2",
     "babel-istanbul": "^0.7.0",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",

